### DISCUSSION TITLE
Delete feature on media library not working properly!

### DISCUSSION THREAD

**Discussion by author31** [strapi/strapi#8465] [Legacy Topics]
I uploaded images to strapi's media library, the id of response body will be increased by each image such as 
http://localhost:1337/upload/files/ => resulted in   { "id": 38,. . }. But what i mean by delete feature not working properly is even though all images in library got deleted but the id of response body didn't update it just keep increasing

Picture of my media library ![image](https://user-images.githubusercontent.com/43377324/97070672-de4dee80-160c-11eb-84ad-ac8b284a5873.png)
Picture of my request 
![image](https://user-images.githubusercontent.com/43377324/97070686-f9206300-160c-11eb-92cd-9466150bf1f6.png)

Is that the error of my implementing ?

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
